### PR TITLE
File Upload Button Style

### PR DESCRIPTION
**Pull request by Jtang-1**
## Product Description

Changes style of "Choose File" button to match other CommCare buttons. More generally, this affects all buttons of an `input` element of `type="file"`. This is a platform level change,.
Before:
Screen Shot 2023-12-20 at 12 56 09 AM

After:
Screen Shot 2023-12-20 at 12 54 25 AM

## Technical Summary

Jira Ticket: https://dimagi-dev.atlassian.net/browse/USH-3937

## Feature Flag

No feature flag

### Safety story

tested locally and on staging. Limited to style change.

### Automated test coverage

no auomated test

### QA Plan

No QA.

### Rollback instructions

- [X] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [X] Risk label is set correctly
- [X] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

File: corehq/apps/hqwebapp/static/hqwebapp/less/_hq/buttons.less [hunk 1]
```diff
@@ -1,8 +1,16 @@
+@import "../../../../../../node_modules/bootstrap/less/buttons.less";
```
Comment (nospame) [inline corehq/apps/hqwebapp/static/hqwebapp/less/_hq/buttons.less:1]: That's a gnarly import. But it does seem to be how we do it in other places.

File: corehq/apps/hqwebapp/static/hqwebapp/scss/commcarehq/_buttons.scss [hunk 2]
```diff
@@ -35,3 +35,20 @@
...
+.btn-default {
+  @include button-variant(white, $call-to-action-mid,  $call-to-action-mid);
+}
+
+.btn-group-xs > .btn, .btn-xs {
+  padding: 1px 5px;
+  font-size: 12px;
+  line-height: 1.5;
+  border-radius: 3px;
+  }
```
Comment (nospame) [inline corehq/apps/hqwebapp/static/hqwebapp/scss/commcarehq/_buttons.scss:48]: What is this set of styles needed for?

Comment (Jtang-1) [reply corehq/apps/hqwebapp/static/hqwebapp/scss/commcarehq/_buttons.scss:48]: `btn-xs` no longer exists for bootstrap 5 so this style recreates it

**Review by nospame — APPROVED** [review]

**Comment by biyeun** [comment]
hey @Jtang-1, for the future, the solution to missing classes for the bootstrap migration should not be to add the missing style. In the case of bootstrap 5, `btn-xs` was renamed to `btn-sm` and has the same functionality. Please remove this addition from the stylesheet. Thanks!

The migration helper script covers `btn-xs` here:
https://github.com/dimagi/commcare-hq/blob/2d8590c91c05aa2b2c96d13ed73d7007b8062488/corehq/apps/hqwebapp/utils/bootstrap/spec/bootstrap_3_to_5.json#L52

cc @nospame

**Comment by biyeun** [comment]
@Jtang-1 see Bootstrap's Migration guide for the full explanation of dropping the `btn-xs` class
https://getbootstrap.com/docs/4.6/migration/#buttons

**Comment by Jtang-1** [comment]
@biyeun Thanks for the explanation, removed that addition in this PR https://github.com/dimagi/commcare-hq/pull/33945

**Pull request merged**